tests/test_permissions.py

```diff
@@ -124,8 +126,9 @@ def test_merge_permissions(permissions_1, permissions_2, expected_permissions,
         dataset=dataset_1,
         data_samples=[train_data_sample_1],
     )
-    traintuple = client_1.add_traintuple(spec).future().wait()
-    assert traintuple.status == assets.Status.done
+    traintuple = client_1.add_traintuple(spec)
+    traintuple = assets.Future(traintuple, client_1).wait()
+    assert traintuple.status == models.Status.done
```

Comment:
the wait method used to raise an error if the status wasn't done, I'm not sure if the behavior has been changed in this PR, but this is the reason why there was no assertion on status.

Reply:
There was one here, you are right it is unnecessary